mmdet/models/losses/iou_loss.py

```diff
@@ -466,3 +520,46 @@ def forward(self,
             avg_factor=avg_factor,
             **kwargs)
         return loss
+
+
+@MODELS.register_module()
+class EIoULoss(nn.Module):
+
+    def __init__(self,
+                 eps=1e-6,
```

Comment:
Docstring is required.

Reply:
Hi, I refer to other existing IoULoss implementation such as DIoULoss, this wrapper class do not need to add docstring， and strdoc should be added to the function `eiou_loss()`.

Reply:
The loss class should also add docstring. And you can simply copy the function's docstring to the class.

Reply:
OK, I have updated the docstring.